internal/cmd/cmd.go

```diff
@@ -92,13 +91,13 @@ $ coder completion fish > ~/.config/fish/completions/coder.fish
 	Run: func(cmd *cobra.Command, args []string) {
 		switch args[0] {
 		case "bash":
-			cmd.Root().GenBashCompletion(os.Stdout)
+			_ = cmd.Root().GenBashCompletion(os.Stdout) // Best effort.
```

Comment:
I would only make the best effort comment once?

Reply:
I actually strongly disagree, it is about the concept, there is an error, it is discarded, we indicate it is on purpose, this should be done each and every time. Ideally with more details, but at the very least with a message showing the intent.

Reply:
I think putting a higher level comment that says "we ignore errors when generating completion because they're not critical and the tool degrades gracefully" is very clear, whereas "best effort" doesn't really capture that decision or background as clearly (particularly when redundant.) 

I think the ignored error itself does a pretty good job of communicating intent, though I agree that some kind of comment is useful.

I'm not strongly committed to this view.